economy/schema.py

```diff
@@ -46,6 +59,31 @@
 from users.models import User
 
 
+class ExpenditureDay(graphene.ObjectType):
+    day = graphene.Date()
+    sum = graphene.Int()
+
+
+class TotalExpenditure(graphene.ObjectType):
+    data = graphene.List(ExpenditureDay)
+    total = graphene.Int()
+
+
+class TotalExpenditureItem(graphene.ObjectType):
+    name = graphene.String()
+    total = graphene.Int()
+    quantity = graphene.Int()
+    average = graphene.Int()
```

Comment:
```suggestion
    average = graphene.Float()
```

This needs to be float, or it breaks